cadc-vos-server/src/test/java/ca/nrc/cadc/vos/server/web/restlet/action/GetDetailedNodeActionTest.java

```diff
@@ -123,6 +123,8 @@ protected void prePerformNodeAction() throws Exception
                 "VIEW/REFERENCE").once();
         expect(queryForm.getFirstValue("uri")).andReturn(null).once();
         expect(queryForm.getFirstValue("limit")).andReturn(null).once();
+        expect(queryForm.getFirstValue("sort")).andReturn(null).once();
```

Comment:
once() is brittle because it is restricting the impl to use local vars instead of calling a method multiple times... use atLeastOnce() instead

Reply:
done